src/js/App/Header/UserToggle.js

```diff
@@ -36,12 +36,16 @@ function buildItems(username, accountNumber = -1, extraItems) {
             rel='noopener noreferrer'>
                 My profile
         </DropdownItem>,
-        <DropdownItem
-            key="User management"
-            href="https://www.redhat.com/wapps/ugc/protected/usermgt/userList.html"
-            target="_blank" rel='noopener noreferrer'>
-                User management
-        </DropdownItem>,
+        <React.Fragment key="user management wrapper">
```

Comment:
No need for empty fragment:
```JSX
[
        <DropdownItem key="Username" isDisabled>
            <dl className='ins-c-dropdown-item__stack'>
                <dt className="ins-c-dropdown-item__stack--header">Username:</dt>
                <dd className="ins-c-dropdown-item__stack--value">{username}</dd>
            </dl>
        </DropdownItem>,
        ...isOrgAdmin ? [
          <DropdownItem
                    key="User management"
                    href="https://www.redhat.com/wapps/ugc/protected/usermgt/userList.html"
                    target="_blank" rel='noopener noreferrer'>
                        User management
                </DropdownItem>
        ] : [],
```

Reply:
Sure thing. I can change it. I was just trying to keep it consistent with https://github.com/RedHatInsights/insights-chrome/blob/cfbd2f1316ed717768e65bad9a8d7b6c23cadc01/src/js/App/Header/UserToggle.js#L22

Reply:
Oh I see, well no need to change it to the spread thing (I just like them more than empty elements) :smile: